src/components/backend-ai-credential-view.ts

```diff
@@ -256,6 +260,34 @@ export default class BackendAICredentialView extends BackendAIPage {
           border-bottom: 1px solid #DDD;
         }
 
+        backend-ai-multi-select {
+          /* override for mwc-list */
+          --select-primary-theme: var(--general-sidebar-color);
+          --select-secondary-theme: var(--general-checkbox-color);
+          --select-background-color: var(#E7EBEE, #efefef);
+          --select-background-border-radius: 5px;
+          --select-box-shadow: 0 1px 3px -1px rgba(0,0,0,60%), 0 3px 12px -1px rgb(200,200,200,80%);
+
+          /* override for selected-area */
+          --select-title-font-size: 10px;
+          --selected-area-border-radius: 5px;
+          --selected-area-border: none;
+          --selected-area-padding: 5px;
+          --selected-area-min-height: 24px;
+          --selected-area-height: 100%;
+
+          /* override for selected-item */
+          --selected-item-font-family: var(--general-font-family);
+          --selected-item-theme-color: #C8CED7;
+          --selected-item-theme-font-color: #182739;
+          --selected-item-unelevated-theme-color: #C8CED7;
+          --selected-item-unelevated-theme-color: #C8CED7;
+          --selected-item-outlined-theme-font-color: black;
+          --selected-item-unelevated-theme-font-color: black;
+          --selected-item-font-size: 14px;
+          --selected-item-text-transform: none;
+        }
```

Comment:
The style block for the `backend-ai-multi-select` is the same as the one defined in the `backend-ai-resource-policy-list.ts`: https://github.com/lablup/backend.ai-webui/pull/1327/files#diff-d61f09d5fee23b684f5d92bbb9987a8c9023fb98e1f639c9f4cfde7ff9d0d124R210-R236

How about take this out as the common/default style?